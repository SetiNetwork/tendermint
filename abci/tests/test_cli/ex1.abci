echo hello
info
prepare_proposal "prepare0x05"
finalize_block "abc"
info
commit
query "abc"
finalize_block "def=xyz" "ghi=123"
commit
query "def"
