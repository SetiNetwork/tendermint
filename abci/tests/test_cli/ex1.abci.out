> echo hello
-> code: OK
-> data: hello
-> data.hex: 0x68656C6C6F

> info 
-> code: OK
-> data: {"size":0}
-> data.hex: 0x7B2273697A65223A307D

> prepare_proposal "prepare0x05"
-> code: OK
-> log: Succeded. Tx: 0x05 action: ADDED
> prepare_proposal "prepare0x05"
-> code: OK
-> log: Succeded. Tx: prepare0x05 action: REMOVED

> finalize_block "abc"
-> code: OK

> info 
-> code: OK
-> data: {"size":1}
-> data.hex: 0x7B2273697A65223A317D

> commit 
-> code: OK
-> data.hex: 0x0200000000000000

> query "abc"
-> code: OK
-> log: exists
-> height: 1
-> key: abc
-> key.hex: 616263
-> value: abc
-> value.hex: 616263

> finalize_block "def=xyz" "ghi=123"
-> code: OK
> finalize_block "def=xyz" "ghi=123"
-> code: OK

> commit 
-> code: OK
-> data.hex: 0x0600000000000000

> query "def"
-> code: OK
-> log: exists
-> height: 2
-> key: def
-> key.hex: 646566
-> value: xyz
-> value.hex: 78797A

